src/s1ap/handlers/attach_icsreq.c

```diff
@@ -49,7 +49,7 @@ get_icsreq_protoie_value(struct proto_IE *value, struct init_ctx_req_Q_msg *g_ic
 	ieCnt++;
 
 	/* E-RABToBeSetupItemCtxtSUReq start */
-	ERABSetup *e_rab = &(value->data[ieCnt].val.E_RABToBeSetupItemCtxtSUReq);
+	erab_setup_item *e_rab = &(value->data[ieCnt].val.erab_to_be_setup_item_ctxt_su_req);
 	/* TODO: Remove hardcoded values. */
 	e_rab->e_RAB_ID = 1;
```

Comment:
Point of discussion.. Radio Bearer Id need not be same as EPS bearer id ?

Reply:
It needs to be same.

The value of the E-RAB ID used at S1 and X2 interfaces to identify an E-RAB allocated to the UE is the
same as the EPS Bearer ID value used at the Uu interface to identify the associated EPS Bearer (and also
used at the NAS layer as defined in TS 36.413 [25]). (36.300  sec:8.2)